src/algebra/ordered_ring.lean

```diff
@@ -14,12 +14,34 @@ variable {α : Type u}
 multiplication with a positive number and addition are monotone. -/
 @[protect_proj]
 class ordered_semiring (α : Type u) extends semiring α, ordered_cancel_add_comm_monoid α :=
+(zero_lt_one : 0 < (1 : α))
 (mul_lt_mul_of_pos_left :  ∀ a b c : α, a < b → 0 < c → c * a < c * b)
 (mul_lt_mul_of_pos_right : ∀ a b c : α, a < b → 0 < c → a * c < b * c)
 
 section ordered_semiring
 variables [ordered_semiring α] {a b c d : α}
 
+lemma zero_lt_one : 0 < (1:α) :=
+ordered_semiring.zero_lt_one
+
+lemma zero_le_one : 0 ≤ (1:α) :=
+zero_lt_one.le
+
+/-- `0 < 2`: an alternative version of `two_pos` that only assumes `linear_ordered_semiring`. -/
```

Comment:
`two_pos` was recently renamed to `zero_lt_two` https://leanprover-community.github.io/mathlib_docs/algebra/ordered_ring.html#zero_lt_two 

Will there be a name clash?